From 7d01c6afc05b53d6c04c5ba16e93898dc97f9067 Mon Sep 17 00:00:00 2001
From: Praveen Arimbrathodiyil <praveen@debian.org>
Date: Fri, 14 Nov 2014 21:44:09 +0530
Subject: [PATCH] bump sidekiq to 3.2.6

From upstream changelog,

    Deprecate delay extension for ActionMailer 4.2+ . [seuros, #1933]
    Poll interval tuning now accounts for dead processes [epchris, #1984]
    Add non-production environment to Web UI page titles [JacobEvelyn, #2004]

Debian already have 3.2.6 and I'd like to keep both versions in sync.
---
 Gemfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -29,7 +29,7 @@ gem 'simple_captcha2', '0.3.2', :require
 
 # Background processing
 
-gem 'sidekiq', '3.2.5'
+gem 'sidekiq', '3.2.6'
 gem 'sinatra', '1.3.3'
 
 # Scheduled processing
